src/api/frame/frame.ts

```diff
@@ -24,12 +24,30 @@ export default class _FrameModule extends Base {
     }
 
     /**
-     * Get a reference to the current frame
+     * Synchronously returns a reference to the specified frame. The frame does not have to exist
+     * @param {string} uuid - uuid of the frame you want to wrap
+     * @param {string} name - name of the frame you want to wrap
+     * @return {_Frame}
+     */
+    public wrapSync(uuid: string, name: string): _Frame {
```

Comment:
We should make this match Window in taht it should take identity as one argument as an obj (done incorrectly originally, should fix that too.)